src/main/host/thread_preload.c

```diff
@@ -203,7 +208,11 @@ pid_t threadpreload_run(Thread* base, gchar** argv, gchar** envv) {
     g_free(envStr);
     g_free(argStr);
 
+    RECORD_TIME(fork_start);
     pid_t child_pid = _threadpreload_fork_exec(thread, argv[0], argv, myenvv);
+    RECORD_TIME(fork_stop);
+
+    FORK_DURATION_ACC += duration_to_seconds(&fork_start, &fork_stop);
```

Comment:
Is this addition to a global variable thread safe? Or are we doing all of the forking *before* creating the worker threads?

Instead of the global, I wonder if we might want to store the fork time counter in the (thread-local) `Worker` object, and use a function like `worker_addForkTime(duration_to_seconds(&fork_start, &fork_stop));` to increment it? Then, each worker would log its fork time before destroying itself.